src/npg_porch/server.py

```diff
@@ -37,32 +41,47 @@
     },
 ]
 
-
 app = FastAPI(
-    title = "Pipeline Orchestration (POrch)",
-    openapi_url = "/api/v1/openapi.json",
-    openapi_tags = tags_metadata,
+    title="Pipeline Orchestration (POrch)",
+    openapi_url="/api/v1/openapi.json",
+    openapi_tags=tags_metadata,
 )
 app.include_router(pipelines.router)
 app.include_router(tasks.router)
 
+templates = Jinja2Templates(directory="src/npg_porch/templates")
+
+
 @app.get(
     "/",
     response_class=HTMLResponse,
     tags=["index"],
-    summary="Web page with links to OpenAPI documentation."
+    summary="Web page with filterable table of tasks and links to OpenAPI "
+            "documentation."
```

Comment:
This is weird

Reply:
In what way is it weird?  Is the weirdness a bad thing?  How can I make it less weird?

Reply:
I'm pretty sure splitting strings like this is fine in python, but I will check. In any case, the second string cannot be a different argument because there is no comma after the first one.

Reply:
I guess it's fine if it's auto-concatenating the strings. You prefer this to `"""`?

Reply:
I think `"""` seems like overkill for two lines, in general...